src/java/Main.java

```diff
@@ -0,0 +1,49 @@
+package java;
+
+public class Main {
+
+    public static void main(String[] args) {
+        Civil AN=new Civil();
+        AN.BusinessClass=false;
+        AN.NumberOfPassengers=20;
+        AN.maksSpeed=700;
+        AN.MinLength=250;
+        AN.power=1000;
+        AN.WingSpan=50;
+        AN.Vyvod();
+        AN.Power();
+        AN.loading();
+        Military MIG= new Military();
+        MIG.maksSpeed=600;
+        MIG.MinLength=200;
+        MIG.power=1500;
+        MIG.EjectionSystems=true;
+        MIG.NumberOfMissiles=10;
+        MIG.Vyvod();
+        MIG.Power();
+        MIG.ejection();
+        MIG.shot();
+        Light_vehicle BMW=new Light_vehicle();
+        BMW.NumberOfSeats=4;
+        BMW.weight=13000;
+        BMW.maksSpeed=250;
+        BMW.NumberOfWheels=4;
+        BMW.carModel=new String("BMW");
```

Comment:
@Hlieb Не нужно создавать новый объект типа `String` можно просто
```suggestion
        BMW.carModel = "BMW";
```